packages/greencheck/src/Entity/Greencheck.php

```diff
@@ -8,9 +8,9 @@
  * TGWF\GreencheckAdminBundle\Entity\Greencheck.
  *
  * @ORM\Table(name="greencheck",indexes={
- *  @ORM\Index(name="green", columns={"green"}),
- *  @ORM\Index(name="url", columns={"url"}),
- *  @ORM\Index(name="datum", columns={"datum"}),
+ *  @ORM\Index(name="greencheck_idx_green", columns={"green"}),
```

Comment:
indexes need to be unique across all postgres tables/relations/indexes. so i changed their names here.